doc/news/_preparation_next_release.md

```diff
@@ -1,43 +1,44 @@
-# 0.9.<<VERSION>> Release
+# 0.9.2 Release
 
-This release did not happen yet.
+Not yet released.
 
-Please update this file within PRs accordingly.
-For non-trivial changes, you can choose to be
-part of the highlighted changes. Please make
-sure to add some short tutorial (checked by
-shell recorder) or asciinema for highlighted items.
+- shortDesc: `elektrad` in Go, KDE and GNOME Integration, Bugfixes and Documentation Improvements
 
-Please add your name at the end of every contribution.
-**Syntax:** _(your name)_
-
-<<`scripts/generate-news-entry`>>
-
-We are proud to release Elektra 0.9.<<VERSION>>.
+We are proud to release Elektra 0.9.2.
 
 ## What is Elektra?
 
 Elektra serves as a universal and secure framework to access
 configuration settings in a global, hierarchical key database.
 For more information, visit [https://libelektra.org](https://libelektra.org).
 
-You can also read the news [on our website](https://www.libelektra.org/news/0.9.<<VERSION>>-release)
+You can also read the news [on our website](https://www.libelektra.org/news/0.9.2-release)
 
 ## Highlights
 
-- <<HIGHLIGHT1>>
-- <<HIGHLIGHT2>>
-- <<HIGHLIGHT3>>
+- `elektrad` rewritten in Go
+- KDE and GNOME Integration
```

Comment:
I would make this the first highlight, unless you plan to make advertising in the Go community. And maybe make two highlights out of it, "bugfixes and documentation" is not really  a highlight :wink:

Reply:
I don't plan to advertise in the Go community. I placed `elektrad` first as this is the probably the biggest change within the release.